applications/ConvectionDiffusionApplication/custom_elements/symbolic_d_convection_diffusion_explicit.cpp

```diff
@@ -0,0 +1,1783 @@
+    const Variable<double>& rVariable,
+    double& Output,
+    const ProcessInfo& rCurrentProcessInfo)
+{
+    KRATOS_TRY;
+
+    auto& r_geometry = this->GetGeometry();
+    const unsigned int local_size = r_geometry.size();
+    VectorType rhs_oss;
+    this->CalculateOrthogonalSubgridScaleSystem(rhs_oss,rCurrentProcessInfo);
+    for (unsigned int i_node = 0; i_node < local_size; i_node++) {
+        #pragma omp atomic
+        r_geometry[i_node].GetValue(rVariable) += rhs_oss[i_node];
+    }
+
+    KRATOS_CATCH("");
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+void SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::CalculateOrthogonalSubgridScaleSystem(
+    VectorType& rRightHandSideVector,
+    const ProcessInfo& rCurrentProcessInfo)
+{
+    KRATOS_TRY;
+
+    const auto& r_geometry = this->GetGeometry();
+    const unsigned int local_size = r_geometry.size();
+
+    // Resize and intialize output
+    if (rRightHandSideVector.size() != local_size)
+        rRightHandSideVector.resize(local_size, false);
+    noalias(rRightHandSideVector) = ZeroVector(local_size);
+
+    // Element variables
+    ElementVariables rVariables;
+    this->InitializeEulerianElement(rVariables,rCurrentProcessInfo);
+
+    // Compute tau
+    this->CalculateTau(rVariables);
+
+    // Execute OSS step
+    this->CalculateOrthogonalSubgridScaleSystemInternal(rVariables,rRightHandSideVector);
+
+    KRATOS_CATCH("");
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template< unsigned int TDim, unsigned int TNumNodes >
+int SymbolicDConvectionDiffusionExplicit<TDim,TNumNodes>::Check(const ProcessInfo &rCurrentProcessInfo)
+{
+    KRATOS_TRY;
+
+    int out = Element::Check(rCurrentProcessInfo);
+    KRATOS_ERROR_IF_NOT(out == 0)
+        << "Error in base class Check for Element " << this->Info() << std::endl
+        << "Error code is " << out << std::endl;
+    return 0;
+
+    KRATOS_CATCH("");
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template <>
+void SymbolicDConvectionDiffusionExplicit<2>::CalculateLocalSystemInternal(
+    ElementVariables& rVariables,
+    MatrixType& rLeftHandSideMatrix,
+    VectorType& rRightHandSideVector)
+{
+    KRATOS_TRY;
+
+    // Retrieve element variables
+    const auto& k = rVariables.diffusivity;
+    const auto& f = rVariables.forcing;
+    const auto& phi = rVariables.unknown;
+    const auto& phi_old = rVariables.unknown_old;
+    const auto& delta_time = rVariables.delta_time;
+    const auto& RK_time_coefficient = rVariables.RK_time_coefficient;
```

Comment:
Why do you need the RK coefficients in here? Shouldn't the inertial terms be managed by the explicit RK scheme?

Reply:
I use them to compute the temporal derivative in the residual. In other words:
![CodeCogsEqn](https://user-images.githubusercontent.com/43809717/93785701-ada22000-fc2e-11ea-8424-4b7d389f36b1.gif)
And depending on the RK step we have a different value for RK_time_coefficient.

Reply:
But shouldn't this be done at the strategy level? Otherwise for each node you are computing it as many times as elements share such node.

Reply:
I think it could be done at the price of saving in the memory one more variable. Would you go for this?

Reply:
After a brief discussion we agreed that in this way is as reasonable as the one I was pointing.

My unique comment is to rename the `RK_time_coefficient` variable to `delta_time_coefficient` or something of the sort to make clear that the element is compatible with any explicit scheme.

Reply:
Ok, done.